google/cloud/security/inventory/inventory_loader.py

```diff
@@ -250,25 +235,22 @@ def main(argv):
     crm_rate_limiter = RateLimiter(max_crm_calls, 100)
 
     pipelines = [
-        {'pipeline': load_projects_pipeline,
-         'status': ''},
-        {'pipeline': load_projects_iam_policies_pipeline,
-         'status': ''},
-        {'pipeline': load_org_iam_policies_pipeline,
-         'status': ''},
+        LoadOrgIamPoliciesPipeline(
```

Comment:
Ack, I am planning to make a sweeping change to all the other pipelines and include them in this PR for merging.